modules/rollback-operator-upgrades.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
```

Comment:
We simplified the rollback for 4.0 to 74.z so that we would not need to do the checks for version. @msugakov IIRC, we can cut out everything from step 3 and down: https://58684--docspreview.netlify.app/openshift-acs/latest/upgrading/upgrade-operator.html?

Reply:
@connorgorman Please ignore this file. I forgot to delete this one and Kerry already worked on the rollback section.